README.md

```diff
@@ -2,4 +2,4 @@
 
 ## a readme to explain nothing
 
-I fixed the misspelled words. This should be good for the sample review. Thanks.
\ No newline at end of file
+jdbsogbwihn gjewo pou gewo jgpjpjgej/gr woj vro
```

Comment:
Please spell these words correctly, thank you!

Reply:
@burgess01 I fixed these changes